openmc/mgxs/mgxs.py

```diff
@@ -2452,7 +2866,7 @@ def __init__(self, domain=None, domain_type=None,
         self._rxn_type = 'nu-scatter'
 
 
-class ScatterMatrixXS(MGXS):
+class ScatterMatrixXS(MatrixMGXS):
```

Comment:
Shouldn't you be able to eliminate the `ScatterMatrixXS` class methods `get_xs(...)`, `get_slice(...)` and `print_xs(...)` since they are defined in the parent `MGXSMatrixXS` class? Also, what about the `get_pandas_dataframe(...)` method? Have you run MGXS notebooks I, II and III?